app/views/admin/settings/working_days_and_hours_settings/show.html.erb

```diff
@@ -27,22 +27,51 @@ See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
 
-<%= toolbar title: t(:label_working_days) %>
-
-<div class="op-toast -warning">
-  <div class="op-toast--content">
-    <p>
-    <%= t("working_days.warning").html_safe %>
-    </p>
-  </div>
-</div>
+<%= toolbar title: t(:label_working_days_and_hours) %>
 
 <%= styled_form_tag(
-      admin_settings_working_days_path,
+      admin_settings_working_days_and_hours_path,
       method: :patch,
       class: 'op-working-days-admin-settings'
     ) do %>
 
+  <section class="form--section"
+           data-application-target="dynamic"
+           data-controller="admin--working-days-and-hours">
+    <fieldset class="form--fieldset">
+      <div class="form--field">
+        <%= setting_text_field :hours_per_day,
+                               type: :number %>
+        <div class="form--field-instructions"><%= t("setting_hours_per_day_explanation") %></div>
+      </div>
+
+      <div class="form--field">
+        <%= setting_text_field :days_per_week,
+                               type: :number,
+                               data: { admin__working_days_and_hours_target: 'daysPerWeekInput',
+                                       action: 'blur->admin--working-days-and-hours#recalculateDaysPerMonth' } %>
+        <div class="form--field-instructions"><%= t("setting_days_per_week_explanation") %></div>
+      </div>
+
+      <div class="form--field">
+        <%= setting_text_field :days_per_month,
+                               type: :number,
+                               data: { admin__working_days_and_hours_target: 'daysPerMonthInput',
+                                       action: 'blur->admin--working-days-and-hours#recalculateDaysPerWeek' } %>
+        <div class="form--field-instructions"><%= t("setting_days_per_month_explanation") %></div>
+      </div>
+
+    </fieldset>
+  </section>
+
+  <div class="op-toast -warning">
+    <div class="op-toast--content">
+      <p>
+        <%= t("working_days.warning").html_safe %>
+      </p>
+    </div>
+  </div>
```

Comment:
This is the commit https://github.com/opf/openproject/pull/15450/commits/adfa1bd7f6863d2f953f25f95cbe9597b514e04d

in case we agree this is where it should be

Reply:
Seen with Parimal during today's frontend meeting:
- the text has been slightly reworded
- the warning is moved on top of the section, so the warning can be read before doing the changes

![image](https://github.com/opf/openproject/assets/176055/d44c5148-0b41-4dac-912d-9021e66006a3)

He also asked if it's possible when clicking "Apply changes" to display the warning message about changed days only if there are actually changed days. I'll do another review to specifically ask for this so we can track it more easily.